app/src/main/java/ar/com/wolox/android/example/ui/home/HomeActivity.kt

```diff
@@ -0,0 +1,43 @@
+package ar.com.wolox.android.example.ui.home
+
+import ar.com.wolox.android.R
+import ar.com.wolox.android.example.ui.home.news.NewsFragment
+import ar.com.wolox.android.example.ui.home.profile.ProfileFragment
+import ar.com.wolox.wolmo.core.activity.WolmoActivity
+import ar.com.wolox.wolmo.core.adapter.viewpager.SimpleFragmentPagerAdapter
+import kotlinx.android.synthetic.main.activity_home.*
+import javax.inject.Inject
+
+/**
+ * Application home;
+ */
+class HomeActivity : WolmoActivity() {
+
+    private lateinit var mSimpleFragmentPagerAdapter: SimpleFragmentPagerAdapter
+
+    @Inject internal lateinit var mNewsFragment: NewsFragment
+    @Inject internal lateinit var mProfileFragment: ProfileFragment
+
+    override fun layout(): Int = R.layout.activity_home
```

Comment:
Here you can omit the type of the declaration. Kotlin supports type inference which means that the compiler find out types from context.
It would look like this: `override fun layout() = R.layout.activity_home` and the Android Studio will put the `: Int` by itself.

Reply:
Removed type declaration